MakeProject/source/fun.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include "include/fun.h"
```

Comment:
Update CMake file so, that you could include in the following way:
#include "fun.h"